skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
Not with HLS as it is DVR. I remember HDS or HD3 can detect and gives out a flag about DVR is available.